package.json

```diff
@@ -5,12 +5,12 @@
   "main": "lib/index",
   "repository": {
     "type": "git",
-    "url": "git://github.com/tommy351/hexo-webui.git"
+    "url": "hexojs/hexo-webui"
```

Comment:
According to [npm document](https://docs.npmjs.com/files/package.json#repository), if the schema is omitted, it is stated that it is recognized as referring from npm.
I propose to include the `github:` schema.